crates/biome_css_analyze/src/utils.rs

```diff
@@ -109,3 +111,41 @@ pub fn is_function_keyword(value: &str) -> bool {
 pub fn is_custom_function(value: &str) -> bool {
     value.starts_with("--")
 }
+
+// Returns the vendor prefix extracted from an input string.
+pub fn vender_prefix(prop: &str) -> String {
+    let re = Regex::new(r"^(-\w+-)").unwrap();
+    let vendor_prefix = re
+        .captures(prop)
+        .map(|caps| caps[0].to_string())
+        .unwrap_or_default();
+
+    vendor_prefix
+}
```

Comment:
I think we don't need `Regex` since the case is really simple and `Re crate is expensive. We can just define the prefixes in an ordered array, and check against them using the binary search.

Reply:
[The `-moz-test` in the test case](https://github.com/biomejs/biome/pull/2655/files#diff-f96866d75ccb4f37de9b47a64f49acac6b951de9aa29048013d3d049bf8c236dR22) also needs to be extracted as a vendor prefix. Since I understand that binary_search cannot perform partial matches, I replaced it with starts_with for the correction.
b0892bdbf0c873cf4526d280b34045fea7acdd49